src/Compilers/CSharp/Test/Symbol/Symbols/Source/NullablePublicAPITests.cs

```diff
@@ -3815,5 +3815,56 @@ void verify(DefaultExpressionSyntax expr, PublicNullableAnnotation expectedAnnot
                 Assert.Equal(expectedState, info.FlowState);
             }
         }
+
+        [Fact]
+        public void GetTypeInfoOnNullableType()
+        {
+            var source = @"
+#nullable enable
+using System.Collections.Generic;
+namespace N
+{
+    class C
+    {
+        void M(C x)
+        {
+            global::N.C c1 = x;
+            global::N.C? c2 = x;
+            N.C c3 = x;
+            N.C? c4 = x;
+            C c5 = x;
+            C? c6 = x;
+            M2(out C c7);
+            M2(out C? c8);
+            foreach (C c9 in M3()) {}
+            foreach (C? c10 in M3()) {}
+        }
+
+        void M2(out C c) => throw null!;
+        IEnumerable<C> M3() => throw null!;
+    }
+}
+";
+            var comp = CreateCompilation(source);
+            comp.VerifyDiagnostics();
+
+            var tree = comp.SyntaxTrees.Single();
+            var model = comp.GetSemanticModel(tree, ignoreAccessibility: false);
+            var identifiers = tree.GetRoot().DescendantNodes().OfType<VariableDeclarationSyntax>().ToList();
```

Comment:
>identifiers [](start = 16, length = 11)

Please assert there are only 12 items in the list

Reply:
6, but yes.

---
In reply to: [366606901](https://github.com/dotnet/roslyn/pull/40950#discussion_r366606901) [](ancestors = 366606901)